starter/src/test/java/dev/openbanking4/spring/security/multiauth/configurers/collectors/PSD2CollectorTest.java

```diff
@@ -127,9 +133,33 @@
     @Before
     public void setUp() {
 
+        // Build a collector that will not return a username if the cert is not an eidas WEB cert.
         this.psd2Collector = PSD2Collector.psd2Builder()
                 .collectorName("psd2-for-test")
-                .usernameCollector(certificatesChain -> certificatesChain[0].getSubjectDN().getName())
+                .usernameCollector((certificatesChain, psd2CertInfo) -> {
+                    String userName = null;
+                    if(psd2CertInfo != null && psd2CertInfo.isPsd2Cert()){
+                        Optional<EidasCertType> eidasCertType = psd2CertInfo.getEidasCertType();
+                        if (eidasCertType.isPresent()) {
+                            if( eidasCertType.get() == EidasCertType.WEB ) {
```

Comment:
Minor - the formatting of the if statements is a bit inconsistent in each of these classes. The formatter dictates a space after the `if`.